src/index.js

```diff
@@ -1,6 +1,32 @@
-/**
-* Put your JavaScript here
-*/
+let state = {
+  todos: [
+    { title: 'First todo', complete: false },
+    { title: 'Second todo', complete: true },
+    { title: 'Third todo', complete: true },
+  ],
+};
+
+document.onload = init();
+
+function init() {
+	const listNode = document.getElementById('list');
+
+	const filterButton = document.getElementById('filter-button')
+	filterButton.addEventListener('click', (event) => {
```

Comment:
I meant literally what was written on the button to provide feedback. Without it, the user won't really know if it's already filtered.